lua/wikis/brawlstars/Infobox/Person/Player/Custom.lua

```diff
@@ -138,18 +142,17 @@ function CustomPlayer:adjustLPDB(lpdbData, args, personType)
 	return lpdbData
 end
 
+---@return Widget?
 function CustomPlayer:createBottomContent()
-	local components = {}
 	if self:shouldStoreData(self.args) and String.isNotEmpty(self.args.team) then
-		local teamPage = Team.page(mw.getCurrentFrame(), self.args.team)
-
-		table.insert(components,
-			Template.safeExpand(mw.getCurrentFrame(), 'Upcoming and ongoing matches of', {team = teamPage}))
-		table.insert(components,
-			Template.safeExpand(mw.getCurrentFrame(), 'Upcoming and ongoing tournaments of', {team = teamPage}))
+		local teamPage = Team.page(mw.getCurrentFrame(),self.args.team)
+		return HtmlWidgets.Fragment{
+			children = {
+				MatchTicker.participant{team = teamPage},
```

Comment:
We should be able to replace this with the player's page, no?
At least for the MatchTicker, UpcomingTournaments would need support for it first.